doc/user_guide/eds.rst

```diff
@@ -525,31 +533,49 @@ Quantification can be applied from the intensities (background subtracted) with
     >>> kfactors = [1.450226, 5.075602] #For Fe Ka and Pt La
     >>> bw = s.estimate_background_windows(line_width=[5.0, 2.0])
     >>> intensities = s.get_lines_intensity(background_windows=bw)
-    >>> weight_percent = s.quantification(intensities, kfactors, plot_result=True)
+    >>> atomic_percent = s.quantification(intensities, method='CL', kfactors)
     Fe (Fe_Ka): Composition = 4.96 weight percent
     Pt (Pt_La): Composition = 95.04 weight percent
 
```

Comment:
if this is now going to give atomic percent as default then it the line above should be updated to give the appropriate atomic percentages rather than weight percentages